### ISSUE TITLE
Nothing happens on export

### ISSUE THREAD

**Issue by Maddara91** [arindam-m/pyslapi#19]
**Describe the bug**
Export not working.
Nothing happens at all.
I've tried some other files and also tried changing the export destination but still nothing...

**To Reproduce**
1. Go to file
2. Click on export skp scene
3. export window pops up
4. click export
5. nothing happens

**System Info**
 - Operating System: w10 20h2
 - Blender Version [e.g. 2.82
 - Importer Version [eg. v.0.20.2]

**Comment by a4jp-com** [comment]
I think it is because of the type of object you have. I noticed some items impost but others don't as well. Hope the update gets released for 2.91.2 so I can test what loads and what doesn't. The last version was working well though.

**Comment by Maddara91** [comment]
But at least, apparently, some things get exported successfully for you. I tried with different objects, even simple models and I tried in 2,79 2,82 and even 2.91. All the same result. Imports perfectly but exports zero things.
Which is intriguing, btw. It ran perfectly for importing on 2.91

**Comment by a4jp-com** [comment]
I just downloaded a model from the 3D warehouse as an extra check and it loaded perfectly. Can you try that?

**Comment by a4jp-com** [comment]
I'm on the latest version of Blender (2.91.2).

**Comment by odil24** [comment]
Try this one too:
https://github.com/RedHaloStudio/Sketchup_Importer/releases/tag/0.22.4

**Comment by theDesuDesu** [comment]
I think there is some confusion here, it's the _export_ that isn't working, not the import which works fine. And I can confirm currently hitting _export_ skp scene does absolutely nothing.

**Comment by arindam-m** [comment]
@theDesuDesu Yes, the export option doesn't do anything. It's just a placeholder, left by the original author of this project - Martijn Berger.

Maybe I could just disable the placeholder (at least, as it shows up in the UI).
